server/sonar-server/src/main/java/org/sonar/server/computation/issue/IssueLifecycle.java

```diff
@@ -92,8 +92,8 @@ public void mergeExistingOpenIssue(DefaultIssue raw, DefaultIssue base) {
     updater.setPastLine(raw, base.getLine());
     updater.setPastLocations(raw, base.getLocations());
     updater.setPastMessage(raw, base.getMessage(), changeContext);
-    updater.setPastEffortToFix(raw, base.effortToFix(), changeContext);
-    updater.setPastTechnicalDebt(raw, base.debt(), changeContext);
+    updater.setPastGap(raw, base.effortToFix(), changeContext);
+    updater.setPastEffort(raw, base.debt(), changeContext);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "debt"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)